### PR TITLE
#164369372 Create custom side navigation component

### PR DESCRIPTION
**Pull request by divinediscipline**
#### What does this PR do?
Creates custom side navigation component

#### Description of Task to be completed?
Create side navigation component with 3 links to profile, articles and statistics page

#### How should this be manually tested?
Clone the repo, then run npm install and npm run start-dev
Then on your browser visit http://localhost:3000/

#### Screenshots (if appropriate)
Screen Shot 2019-03-13 at 3 29 02 PM

### CODE REVIEW

**Review by omenkish — DISMISSED** [review]

**Review by primuse — DISMISSED** [review]

File: src/components/common/likeme.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg width="31" height="30" viewBox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (p-funky) [inline src/components/common/likeme.jsx]: Why is this file named with `.jsx` extension?

Comment (divinediscipline) [reply src/components/common/likeme.jsx]: That file looks strange. I have checked and it should not even be there at all. I did not even create it. Likely an error when merging. I will go ahead and delete it.

Comment (p-funky) [reply src/components/common/likeme.jsx]: If it came about by a merge then that means it was existing elsewhere on the codebase. Kindly confirm with this others that it is okay to delete it.

**Review by EfeAgare — APPROVED** [review]

File: src/components/common/likeme.jsx (hunk 1 above)
Comment (divinediscipline) [reply src/components/common/likeme.jsx]: Yes, I did confirm with my teammates. It is safe to be deleted and I have done so.

**Review by ozone4real — APPROVED** [review]
LGTM

**Pull request merged**